portality/crosswalks/article_crossref_xml.py

```diff
@@ -0,0 +1,304 @@
+import os
+
+from portality.core import app
+from lxml import etree
+import re
+from portality.bll import exceptions
+from portality.crosswalks.exceptions import CrosswalkException
+from portality import models
+
+NS = {'x': 'http://www.crossref.org/schema/4.4.2', 'j': 'http://www.ncbi.nlm.nih.gov/JATS1'}
+
+
+class CrossrefXWalk(object):
```

Comment:
Just to check, was the conversion from Crossref XML to DOAJ article model straightforward and obvious?  Is it worth us documenting it and putting it in front of the DOAJ team to check that they're happy with the conversion?

Reply:
I was quite obvious. I may explain it during the tech call maybe?